### ISSUE TITLE
Q: clear logo for tv and movies when video playback is via add-on

### ISSUE THREAD

**Issue by mgonzales71** [marcelveldt/xbmc.skin.titan#12]
I noticed that xbmc doesn't provide an easy way to get clear art / clear logo when video playback is from an addon but I was looking the $INFO variables and noticed you can get folder and full filename path info - so I googled and found some code 

specifically - this:

$INFO[Player.FolderPath,,../logo.png] to display the clear logo of tv and movies

would this work?

here is blob of code I found:

(from: https://github.com/lumened/skin.flux/blob/master/720p/script.pseudotv.TVOverlay.xml) 

	
<control type="group">
<visible>VideoPlayer.Content(episodes) + Window.IsVisible(FullscreenVideo)</visible>
<animation effect="fade" time="200">VisibleChange</animation>

<control type="image" id="990">
<description>Use TV Clearart if available(episodes in season folders)</description>
<posx>8</posx>
<posy>495</posy>
<width>275</width>
<height>145</height>
<texture fallback="blank.png">$INFO[Player.FolderPath,,../clearart.png]</texture>
<aspectratio>keep</aspectratio>
</control>
<control type="image" id="991">
<description>Use TV Clearart if available(episodes in tv show folder)</description>
<posx>8</posx>
<posy>495</posy>
<width>275</width>
<height>145</height>
<texture fallback="blank.png">$INFO[Player.FolderPath,,clearart.png]</texture>
<aspectratio>keep</aspectratio>
</control>
<control type="group">
<visible>StringCompare(Control.GetLabel(990),blank.png) + StringCompare(Control.GetLabel(991),blank.png)</visible>
<control type="image" id="992">
<description>Otherwise try TV logo if available(episodes in season folders)</description>
<posx>8</posx>
<posy>565</posy>
<width>276</width>
<height>75</height>
<texture fallback="blank.png">$INFO[Player.FolderPath,,logo.png]</texture>
<aligny>bottom</aligny>
<aspectratio align="left">keep</aspectratio>
</control>
<control type="image" id="993">
<description>Otherwise try TV logo if available(episodes in tv show folder)</description>
<posx>8</posx>
<posy>565</posy>
<width>276</width>
<height>75</height>
<texture fallback="blank.png">$INFO[Player.FolderPath,,../logo.png]</texture>
<aligny>bottom</aligny>
<aspectratio align="left">keep</aspectratio>
<visible>StringCompare(Control.GetLabel(992),blank.png)</visible>
</control>
<control type="group">
<visible>StringCompare(Control.GetLabel(992),blank.png) + StringCompare(Control.GetLabel(993),blank.png)</visible>
<control type="image" id="994">
<description>Otherwise try TV folder.jpg if available(episodes in season folders)</description>
<posx>80</posx>
<posy>460</posy>
<width>132</width>
<height>180</height>
<texture fallback="blank.png" diffuse="mask-rect.png">$INFO[Player.FolderPath,,folder.jpg]</texture>
<aligny>bottom</aligny>
<aspectratio align="left">keep</aspectratio>
</control>
<control type="image" id="995">
<description>Otherwise try TV folder.jpg if available(episodes in tv show folder)</description>
<posx>80</posx>
<posy>460</posy>
<width>132</width>
<height>180</height>
<texture fallback="blank.png" diffuse="mask-rect.png">$INFO[Player.FolderPath,,../folder.jpg]</texture>
<aligny>bottom</aligny>
<aspectratio align="left">keep</aspectratio>
<visible>StringCompare(Control.GetLabel(994),blank.png)</visible>
</control>
</control>
</control>
</control>

**Comment by marcelveldt** [comment]
No this won't work. The addons simply don't have the clearlogo to provide to Kodi.

The code example is from previous versions of Kodi (pre frodo as I recall) and a bad way to handle this. The art is available as listitem properties which is a much better way.

What you can do is just add the shows/movies from those addons to your library, that way the content will be enriched my the metadata scanners of Kodi and so will the clearart be picked up.

**Comment by mgonzales71** [comment]
i knew it would be a hack - even when you use library integration for add-on video - it will not pass clear art or clear logo for episodes or shows.

no worries - I was just trying to think a way around that limitation since I have never seen a skin that has done that yet.

I tried to use $INFO[Player.FolderPath,,../logo.png] but it doesn't work 

$INFO[Player.FolderPath] doesnt seem to contain any value when I tried to just display it as text.

I will close this again.  thanks for the reply.  I am enjoying learning if even the basics of skinning.
